web-common/src/features/dashboards/time-series/utils.ts

```diff
@@ -44,20 +43,19 @@ export function prepareTimeSeries(
   original,
   comparison,
   timeGrainDuration: string,
-  zone: string,
 ) {
   return original?.map((originalPt, i) => {
     const comparisonPt = comparison?.[i];
 
-    const ts = adjustOffsetForZone(originalPt.ts, zone);
```

Comment:
For the charts, we need to have this offset. Javascript assumes the dates are in the user's timezone. We need to account for that across our charts

Reply:
could you elaborate what is broken in the screenshot?